README.md

```diff
@@ -1 +1,36 @@
 # hw01
+
+My name is Leyi Kang. I am a master student studying psychology in the Social Sciences Division. I just graduated from the University of Rochester this May, and I look forward to delving deeper into psychology research on child and adolescent psychopathology at UChicago. My long-term goal is to apply for either clinical or developmental psychology PhD after I graduate from here. 
+
+
+# Header 1
+
+## Header 2
+
+### Header 3
+
+#### Header 4
+
+*This sentence will be italic.* or _This will also be italic._
+
+**This sentence will be bold.** or __This will also be bold.__
+
+__This will be *both*.__
+
+* Unordered List
+* Colors
+  + blue
+  + red
+  + yellow
+  
+1. Ordered list
+2. Animals
+   1. horse
+   2. dolphin 
+
+
+Here's a picture of the sea: ![Sea](https://techcrunch.com/wp-content/uploads/2017/05/gettyimages-580833893.jpg?w=990&crop=1)
```

Comment:
I love how you got the image function to work! I couldn't figure it out for the life of me.

Reply:
Beautiful picture!